compiler/scripts/extract-and-check

```diff
@@ -1,25 +1,27 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -e
 
-DIR=$(mktemp -d jasminXXXXXX)
+ROOT_DIR=tests/results/extraction
+DIR=$ROOT_DIR/${2%.jazz}/$1
+ECLIB=$(dirname $0)/../../eclib
 EC=${DIR}/jasmin.ec
 CT=${DIR}/jasmin_ct.ec
 
-if [ "$1" = "arm" ]
-then
-	shift
+if [ -d "${DIR}" ]; then rm -r ${DIR}; fi
+mkdir -p $DIR
+
+if [ "$1" = "arm" ]; then
 	ARCH="-arch arm-m4"
 else
 	ARCH="-arch x86-64"
 fi
 
-trap "rm -r ${DIR}" EXIT
-
 set -x
+set -o xtrace
 
-$(dirname $0)/../jasminc ${ARCH} -oecarray ${DIR} -oec ${EC} "$@"
-$(dirname $0)/../jasminc ${ARCH} -oecarray ${DIR} -CT -oec ${CT} "$@"
+$(dirname $0)/../jasminc ${ARCH} -oecarray ${DIR} -oec ${EC} "$2"
+$(dirname $0)/../jasminc ${ARCH} -oecarray ${DIR} -CT -oec ${CT} "$2"
 
-easycrypt ${ECARGS} ${EC}
-easycrypt ${ECARGS} ${CT}
+easycrypt -I Jasmin:${ECLIB} ${ECARGS} ${EC}
+easycrypt -I Jasmin:${ECLIB} ${ECARGS} ${CT}
```

Comment:
Together with this change, you can clean the CI scripts (e.g.: https://github.com/jasmin-lang/jasmin/blob/main/.gitlab-ci.yml#L184).

Reply:
Done (can be done in only one place in the CI file: the other places to not run the `extract-and-check` script).